src/routes/Home/SearchMedia.tsx

```diff
@@ -107,48 +110,21 @@ const SearchMedia = (): ReactElement => {
         </div>
       </SearchMediaLayout>
       <SearchResultLayout>
-        <div className="search-results">
-          {!data && loading ? (
-            <Loading></Loading>
-          ) : error ? (
-            <span className="search-error">error</span>
-          ) : !showResults ? (
-            <span></span>
-          ) : data ? (
-            data?.searchMediaByTitle?.nodes?.map((media) => {
-              return (
-                <SearchResults key={media?.id}>
-                  <Link
-                    className="media-link"
-                    to={`/${media?.type}/${media?.id}`}>
-                    <Poster
-                      className="poster-image"
-                      style={{
-                        backgroundImage:
-                          'url(' + media?.posterImage.original.url + ')',
-                      }}
-                    />
-                    <Title className="media-title">
-                      {media?.titles.canonical}
-                    </Title>
-                    {/* TODO: Change media.type to media.subtype */}
-                    <SubtypeTag className="subtype-tag">
-                      {media?.type}
-                    </SubtypeTag>
-                    <div className="season-date">
-                      {seasonYear(media?.startDate)}
-                    </div>
-                    <div className="search-description">
-                      <span>{media?.description.en}</span>
-                    </div>
-                  </Link>
-                </SearchResults>
-              );
-            })
-          ) : (
-            <div></div>
-          )}
-        </div>
+        {loading ? (
+          <Loading></Loading>
+        ) : error ? (
+          <span className="search-error">error</span>
+        ) : !searchIsRendered ? (
+          <></>
```

Comment:
why do we do this?

Reply:
To remove the search results if you remove your search query entirely and leave the field blank. Better UX imo.